packages/org.openapis.v3/MediaType.pkl

```diff
@@ -0,0 +1,46 @@
+//===----------------------------------------------------------------------===//
+// Copyright © 2024 Apple Inc. and the Pkl project authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//===----------------------------------------------------------------------===//
+/// Each Media Type Object provides schema and examples for the media type identified by its key.
+module org.openapis.v3.MediaType
+
+import "Schema.pkl"
+import "Encoding.pkl"
+import "Reference.pkl"
+
```

Comment:
It's nice to offer a typealias for some common media types and use them elsewhere (see my suggestions in other files)

```suggestion

typealias MediaTypeName =
  "application/json"
  |"application/xml"
  |"application/x-www-form-urlencoded"
  |"multipart/form-data"
  |"text/plain; charset=utf-8"
  |"text/html"
  |String
  
```

Reply:
I also considered this. My only worry is that the list of possible media types is quite extensive (and infinitely extensible to boot):

https://www.iana.org/assignments/media-types/media-types.xhtml

I didn't want to make too many assumptions about what media types people might be using.